scripts/remove_work/run.py

```diff
@@ -0,0 +1,300 @@
+        aws_access_key_id=credentials["AccessKeyId"],
+        aws_secret_access_key=credentials["SecretAccessKey"],
+        aws_session_token=credentials["SessionToken"],
+        region_name="eu-west-1"
+    )
+
+
+def remove_image_from_es_indexes(catalogue_id):
+    print("*** Removing the image from our Elasticsearch indexes")
+
+    ecs_client = platform_client("ecs")
+    ssm_client = platform_client("ssm")
+
+    # AWLC: Yes, it would be more robust if we got this config by checking the
+    # ingestor task definition to see what the current parameters are, but that's
+    # such a faff with the namespace that I literally CBA.
+    print("··· Reading Elasticsearch config for the ingestor (write credentials)")
+    es_username = ssm_client.get_parameter(
+        Name="/aws/reference/secretsmanager/catalogue/ingestor/es_username",
+        WithDecryption=True,
+    )
+    es_password = ssm_client.get_parameter(
+        Name="/aws/reference/secretsmanager/catalogue/ingestor/es_password",
+        WithDecryption=True,
+    )
+    es_auth = (es_username["Parameter"]["Value"], es_password["Parameter"]["Value"])
+
+    print("··· Getting the task definitions for the catalogue API")
+    resp = ecs_client.list_services(cluster="catalogue-api")
+    service_arns = resp["serviceArns"]
+    assert len(service_arns) == 2
+
+    resp = ecs_client.describe_services(cluster="catalogue-api", services=service_arns)
+    services = resp["services"]
+    assert len(services) == 2
+    task_definitions = [service["taskDefinition"] for service in services]
+
+    miro_id = None
+
+    print("··· Reading Elastic Cloud config for the catalogue API (read credentials)")
+    for td in task_definitions:
+        resp = ecs_client.describe_task_definition(taskDefinition=td)
+        container_definitions = resp["taskDefinition"]["containerDefinitions"]
+        assert len(container_definitions) == 2
+
+        app_containers = [cd for cd in container_definitions if cd["name"] == "app"]
+        assert len(app_containers) == 1
+
+        app_env_vars = {e["name"]: e["value"] for e in app_containers[0]["environment"]}
+
+        app_secrets = {s["name"]: s["valueFrom"] for s in app_containers[0]["secrets"]}
+
+        for name, value_from in app_secrets.items():
+            resp = ssm_client.get_parameter(Name=value_from, WithDecryption=True)
+            app_secrets[name] = resp["Parameter"]["Value"]
+
+        # 3. Once we have the config and password, we can remove the work from
+        # the Elasticsearch index (if present).
+        es_host = "%s://%s:%s/" % (
+            app_secrets["es_protocol"],
+            app_secrets["es_host"],
+            app_secrets["es_port"],
+        )
+
+        for index_name in (app_env_vars["es_index_v1"], app_env_vars["es_index_v2"]):
+            print("··· Looking up %s in index %s" % (catalogue_id, index_name))
+            resp = requests.get(
+                f"{es_host}{index_name}/_doc/{catalogue_id}",
```

Comment:
Ah you have used it here :tada: (see my comment https://github.com/wellcometrust/catalogue/pull/90/files#r309175621)

Reply:
Thanks! This was largely a copy . paste job (but nice to clean it up).